cmake/options.cmake

```diff
@@ -21,6 +21,9 @@ option(OSQUERY_NO_DEBUG_SYMBOLS "Whether to build without debug symbols or not,
 
 option(BUILD_TESTING "Whether to enable and build tests or not")
 
+# TODO: Revert this to 'facebook' before we merge!
+set(OSQUERY_THIRD_PARTY_SOURCE "source;pre-built;facebook" CACHE STRING "Sources used to acquire third-party dependencies")
```

Comment:
Does it make sense to update the other variables to follow this naming convention? `THIRD_PARTY_REPOSITORY_URL` becomes `OSQUERY_THIRD_PARTY_URL`?

Reply:
Can `facebook` become `legacy` or similar?

Reply:
I didn't call it 'legacy' as it's the main source for Buck libraries (and it's still actively used), and my aim is to stay 100% compatible